src/node-jspm-jasmine.js

```diff
@@ -63,21 +58,23 @@ export function runTests(opts, errCallback = function() {}) {
 			if (!Array.isArray(coverageFilesGlobs) || coverageFilesGlobs.length === 0) {
 				console.log(`Not capturing coverage because opts.coverage.files is not a valid array of globs. Also try the '--coverage-files <glob>' command line opt`);
 			}
-			opts.coverage.files.forEach( pattern => {
+			opts.coverage.files.forEach(pattern => {
 				glob.sync(path.join(process.cwd(), pattern))
 				.forEach(file => {
+
 					let isSpec = false;
 					specFiles.forEach(specFileGlob => {
-						isSpec = isSpec || minimatch(file, specFileGlob);
+						isSpec = isSpec || minimatch(file, path.join(process.cwd(), specDir, specFileGlob));
```

Comment:
The `importTestFiles` function that has been there before my first contribution assumes that `spec_dir` exists, so I followed suit. And looking at this [commit](https://github.com/jasmine/jasmine-npm/pull/70/commits/eaa2b3679f123c39b1bc7e4495e1d16190c04cc4), it seems like it was a requirement up until recently. Doing `jasmineConfig.spec_dir || ''` similarly to what they've done, can work.

Btw, I screwed up on the description above, the extended comment was towards issue #17 not #26, it's fixed now.